### ISSUE TITLE
Make it so @mention of users doesn't occur in code blocks/snippets

### ISSUE THREAD

**Issue by michael-tharrington** [thepracticaldev/dev.to#9077]
**Is your feature request related to a problem? Please describe.**

Yes! Occasionally users are writing code that inadvertently @mentions a user. 

For example, a user may choose the username "import" which would cause issues for anyone trying to write @import in their CSS code. (Shoutout to @vaidehijoshi for this example!)

**Describe the solution you'd like**

When an author writes @anything in code block or code snippets, we should make it so that the @mention functionality doesn't function to avoid interference with the code.

**Describe alternatives you've considered**

We could add words like "import" to our reserved names list.

**Comment by rhymes** [comment]
Thanks for the issue! We'll take your request into consideration and follow up if we decide to tackle this issue.

To our amazing contributors: issues labeled `type: bug` are always up for grabs, but for feature requests, please wait until we add a `ready for dev` before starting to work on it.

To claim an issue to work on, please leave a comment. If you've claimed the issue and need help, please ping @thepracticaldev/oss and we will follow up within 3 business days.

For full info on how to contribute, please check out our [contributors guide](https://github.com/thepracticaldev/dev.to/blob/master/CONTRIBUTING.md).

**Comment by rhymes** [comment]
It happened to me when my `@rhymes` appeared in a liquid tag inside the comment

See https://forem.dev/link2twenty/comment/nd

**Comment by lsparlin** [comment]
I'd like to work on this

**Comment by mikeydorje** [comment]
Closing as resolved. PR #15342 was merged in November 2021 fixing @mentions in code blocks and liquid tags.

If @mentions still trigger in code blocks, please open a new issue with reproduction details.
